english/chocopool/choco.md

```diff
@@ -0,0 +1,10 @@
+  Now you are in the chocolate pool, you are wondering if there are any 
+living creatures in that pool. You keep swimming and swimming.However,you 
+can't find out how deep that pool is.The pool is so wide that you can't
+even guess its width or its length. Suddenly, you hear a strange voice.
+You drink some chocolate to keep you calm, but you can't stop trembling.
+what do you think about this voice??
+
+1.Is it a [bananaboat](bananaboat/bboat.md) floating on the pool surface?
+
+2.Is it the sound of the [hazelnults](hazelnut/hazel.md) terittory?
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.